src/domain/metadata/builders/MetadataPayloadBuilder.ts

```diff
@@ -115,9 +121,14 @@ export class MetadataPayloadBuilder {
             categoryCombos,
             categoryOptions,
             categoryOptionCombos,
+            visualizations,
             ...rest
         } = metadataWithoutDuplicates;
 
+        const visualizationsWithRows = visualizations
+            ? await this.addRowsToVisualizations(originInstance, visualizations as Visualization[])
```

Comment:
[nothing to do] It's unfortunate that we need this type-unsafe assertion, but I see that's how it's done in all the app